src/Transport/Entities/FavouritePlace.php

```diff
@@ -13,6 +13,7 @@ class FavouritePlace implements LabeledWaypointInterface
     protected $geolocation;
     protected $stops = [];
     protected $permanent = false;
+    protected $icon;
```

Comment:
Icon is an implementation detail of the client UI. The domain only knows that there's a category (which may include an enum and a label).